src/toolbar/components/toolbar-menu/toolbarComponent.ts

```diff
@@ -119,6 +126,25 @@ export class ToolbarController {
     return ToolbarType.BUTTON_TWO_STATE;
   }
 
+  public collapseButtons() {
+    let customButtonsIndex;
+    if (this.toolbarItems) {
+      if (this.toolbarViews && this.toolbarViews.length > 0) {
+        customButtonsIndex = this.toolbarItems.length - 2;
+      } else {
+        customButtonsIndex = this.toolbarItems.length - 1;
+      }
+        this.toolbarItems[customButtonsIndex] =
```

Comment:
surprising indent

Reply:
The line was too long so I split the assignment to new line. But this might look better
```javascript
this.toolbarItems[customButtonsIndex] = ToolbarController
  .createKebabFromItems(this.toolbarItems[customButtonsIndex]);
```

Reply:
Oh, the problem is not what happens after that `=`.

It's where the line starts..